.travis.yml

```diff
@@ -18,9 +18,7 @@ addons:
     token:
       secure: $SONAR_SECURITY_TOKEN
     branches:
-      - master
-      - develop
-      - refactor
+      - .* 
```

Comment:
Changed branches so it analyzes all branches. Thsi was the only comment that needed to be addressed before a merge. All other comments can be addressed later or in the next sprint because they require some refactoring.